tools/editor/project_manager.cpp

```diff
@@ -1133,18 +1133,20 @@ void ProjectManager::_files_dropped(StringArray p_files, int p_screen) {
 			memdelete(dir);
 		}
 		if (confirm) {
-			multi_scan_ask->set_text(TTR("Are you sure to scan " + itos(multiple_folders.size()) + " folders?"));
+			multi_scan_ask->get_ok()->disconnect("pressed", this, "_scan_multiple_folders");
+			multi_scan_ask->get_ok()->connect("pressed", this, "_scan_multiple_folders", varray(folders));
```

Comment:
`_scan_multiple_folders(StringArray p_files)` with array argument looks better and API ready :). Can I keep rebind here and passing files in cconnect args?